### PR TITLE
feat(a11y): add title and description to icons

### LINKED ISSUE #31: Make icons accessible
Make icons accessible by following the guidelines in [this article](https://www.sitepoint.com/tips-accessible-svg/)

### PR DESCRIPTION
**Pull request by simeonoff**

### CODE REVIEW

File: src/a11y-meta.yml [hunk 1]
```diff
@@ -184,15 +202,33 @@
 "console":
     title: "Terminal Icon"
     description: "A picture depicting a computer terminal."
+"construction-cone":
+    title: "Construction Cone Icon"
+    description: "A picture depicting a construction cone."
+"construction-project":
+    title: "Construction Project Icon"
+    description: "A picture depicting a construction project."
```
Comment (StefanIvanov) [inline src/a11y-meta.yml]: mention blueprint or architectural plan

File: src/a11y-meta.yml [hunk 2]
```diff
@@ -115,12 +115,27 @@
 "breastfeeding":
     title: "Breast-feeding Icon"
     description: "A picture depicting a woman breastfeeding her child."
+"brick-wall":
+    title: "Brick Wall Icon"
+    description: "A picture depicting a wall with bricks."
```
Comment (StefanIvanov) [inline src/a11y-meta.yml]: wall made of bricks sounds better to me

File: src/a11y-meta.yml [hunk 3]
```diff
@@ -334,6 +376,15 @@
...
     description: "A picture showing the euro and dollar symbols encircled in rotating arrows."
+"forklift-truck":
+    title: "Forklift Truck Icon"
+    description: "A picture depicting a forklift truck."
+"foundation":
+    title: "Foundation Icon"
+    description: "A picture depicting a foundation built with bricks."
+"fountain":
+    title: "Fountain Icon"
+    description: "A picture depicting a fountain."
```
Comment (StefanIvanov) [inline src/a11y-meta.yml]: water fountain

File: src/a11y-meta.yml [hunk 4]
```diff
@@ -571,6 +634,9 @@
 "oil":
     title: "Oil Icon"
     description: "A picture depicting an oil barrel."
+"oil-platform":
+    title: "Oil Platform Icon"
+    description: "A picture depicting an oil platform."
```
Comment (StefanIvanov) [inline src/a11y-meta.yml]: add "in the sea" at the end

File: src/a11y-meta.yml [hunk 5]
```diff
@@ -637,6 +718,9 @@
 "pulse":
     title: "Pulse Icon"
     description: "A picture depicting a cardiogram."
+"radiator":
+    title: "Radiator Icon"
+    description: "A picture showing a radiator in a house."
```
Comment (StefanIvanov) [inline src/a11y-meta.yml]: "A picture showing a radiator used for heating."

File: src/a11y-meta.yml [hunk 6]
```diff
@@ -649,6 +733,9 @@
 "responsive-layout":
     title: "Responsive Layout Icon"
     description: "A picture depicting a mobile phone in front of a laptop computer."
+"restricted-area":
+    title: "Restricted Area Icon"
+    description: "A picture depicting a restricted area."
```
Comment (StefanIvanov) [inline src/a11y-meta.yml]: can we add something like being blocked by a barrier or whatever the thing is called?

File: src/a11y-meta.yml [hunk 7]
```diff
@@ -661,24 +748,42 @@
...
     description: "A picture depicting a blood drop with the plus sign in the middle."
+"road-roller":
+    title: "Road Roller Icon"
+    description: "A picture depicting a road roller machine."
 "rod-of-asclepius":
     title: "Rod of Ascelepius Icon"
     description: "A picture depicting a snake twisted around a poll."
+"rulers":
+    title: "Rulers Icon"
+    description: "A picture depicting a different measuring tools."
```
Comment (StefanIvanov) [inline src/a11y-meta.yml]: remove the "a"

File: src/a11y-meta.yml [hunk 8]
```diff
@@ -751,6 +874,15 @@
 "system-update":
     title: "System Update Icon"
     description: "A picture depicting a computer screen with a reload symbol in the middle."
+"table-saw-blade":
+    title: "Table Saw Blade Icon"
+    description: "A picture depicting a table saw blade."
+"tap-water":
+    title: "Tap Water Icon"
+    description: "A picture depicting a tap water."
```
Comment (StefanIvanov) [inline src/a11y-meta.yml]: Remove the "a"

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

File: src/a11y-meta.yml (hunk 8 above)
Comment (sdimchevski) [reply src/a11y-meta.yml]: done

File: src/a11y-meta.yml (hunk 7 above)
Comment (sdimchevski) [reply src/a11y-meta.yml]: done

File: src/a11y-meta.yml (hunk 6 above)
Comment (sdimchevski) [reply src/a11y-meta.yml]: done

File: src/a11y-meta.yml (hunk 5 above)
Comment (sdimchevski) [reply src/a11y-meta.yml]: done

File: src/a11y-meta.yml (hunk 4 above)
Comment (sdimchevski) [reply src/a11y-meta.yml]: done

File: src/a11y-meta.yml (hunk 3 above)
Comment (sdimchevski) [reply src/a11y-meta.yml]: done

File: src/a11y-meta.yml (hunk 2 above)
Comment (sdimchevski) [reply src/a11y-meta.yml]: done

File: src/a11y-meta.yml (hunk 3 above)
Comment (StefanIvanov) [reply src/a11y-meta.yml]: I still don't see the word water in the description

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

**Review by StefanIvanov — APPROVED** [review]

**Review by aatanasov2 — APPROVED** [review]

**Pull request merged**